recipe/meta.yaml

```diff
@@ -1,35 +1,40 @@
 {% set name = "google-api-core" %}
-{% set version = "2.2.2" %}
+{% set version = "2.10.1" %}
+{% set sha256 = "e16c15a11789bc5a3457afb2818a3540a03f341e6e710d7f9bbf6cde2ef4a7c8" %}
 
 package:
-  name: {{ name|lower }}-split
+  name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 97349cc18c2bb2415f64f1353a80273a289a61294ce3eb2f7ce682d251bdd997
+  sha256: {{ sha256 }}
 
 build:
   number: 0
-  noarch: python
-  skip: True      # [s390x]
+  # noarch: python
```

Comment:
We do not need to skip s390x for the base package—only for the `grcp` variants.

Reply:
We don't even need to skip for the grcp variants anymore. All platforms work now